VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,71 @@
+
+The TSC is constituted by maintainers because the goal is to empower "working" individuals, not companies that "pay". In other words, it provides equal power to individual and corporate contributors.
+
+The TSC's system of operation is by voting. An issue or debate is posted, and the members vote asynchronously and publicly on their decision. The value of the vote of all contributors is equal, thus ensuring the project that there is no risk of one company having a majority of the voters. That is why there is a rule that only 1/4 of the voters can be affiliated with the same company.
+
+## Putting theory into practice
+
+Summing up, AsyncAPI is committed to democratic management based on two principles: horizontality in decision making and transparency in project management. From the beginning of the specification, AsyncAPI has relied on collective effort and group identity. The means are as crucial as the goal. Responsibilities are shared, understanding that a community can only achieve success jointly.
+
+For doing so, AsyncAPI has several tools that ensure transparency within the project’s open-governance management. Let's see how theory is put into practice:
+
+- **Economy**: all the financial management of the project is managed through Open Collective. It’s an online crowdfunding platform for communities that provides the necessary tools to raise and share the finances raised with complete transparency. You can find out where the money comes from in this space and its use.
+
+The TSC makes decisions on financial expenses through voting.
+
+- **Communication**: To ensure that the principle of transparency governs communication, everything is said and done in public, no matter how insignificant or essential it may be. The community uses GitHub as the first communication and work platform. All decisions are taken there. Secondly, Slack is used as a means of sync communication enablement and keeping the closer bond with the community on daily basis.
+
+- **Meetings**: There are different meeting points where the community can meet, propose or discuss any topic related to AsyncAPI. Any interested person can join these meetings and raise any thoughts.
+
+  Even though there are synchronous meetings, there is a written rule that no decision making is done during them. After something is discussed on a        meeting, the community needs to have time to react, watch the recording and decide. Is then when decisions can be finally made, asynchronously.
+
+- **Decision-making**: Decisions are usually discussed on GitHub through issues, PRs, and discussions. For higher-level decision making or when the maintainers cannot find consensus, the Technical Steering Committe is responsible for the oversight of the AsyncAPI Initiative. 
```

Comment:
What do you mean by *higher level*? Sounds a bit contradictory with the *horizontal model*

Reply:
I'm referring to the making of important decisions. I don't think it's contradictory since the TSC is a collective made up of maintainers who are there democratically. It is a fundamental axis for the balance of the horizontality of the project.